profiles/features/musl/package.use.mask

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2015 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+# See bug #604542
+net-misc/iputils idn
+
+# See bug #504200
+sys-devel/gcc sanitize vtv
+
+# llvm's sanitizers are also incompatible with musl
+sys-devel/llvm sanitize
+
+# These cause collisions with <libintl.h>
+# even with --without-included-gettext
+sys-devel/gettext nls
+sys-fs/e2fsprogs nls
+
+# Broken
+dev-vcs/git gpg
```

Comment:
Wasn't that fixed with the stabilization of >gnupg-2.2?

Reply:
yes it was, thanks for catching that.